docker-compose/edge-compose.yml

```diff
@@ -51,19 +65,26 @@ services:
     command: >-
       --pop_id              ${POP_ID:-mypop}
       --server_id           ${SERVER_ID:-myserver}
-      --core_host           ${DIST_OR_CORE_HOSTS:-dist}
+      # Below must be changed to 'localhost' if container is run in 'host' mode.
+      --core_host           ${DIST_HOSTS:-dist}
+      # If you used the BOOTSTRAP UI to perform initial configuration, this can
+      # be left alone.  Otherwise, change this to your DHCP service definition ID
+      --dhcp_service_def_id 2
       --enable_ops_metrics  true
```

Comment:
Perhaps we ought to provide example for `dhcp_peers` when doing an HA DHCP setup? At least that makes sense in `edge-compose.yml`, I think and that follows the precedent you made with `control-compose.yml` referencing `data_peers` and whatnot.

Reply:
```
#      --dhcp_ha             enabled
#      --dhcp_peers          dhcp2
#      --dhcp_is_primary     true
#      --dhcp_heartbeat_delay     500
#      --dhcp_max_response_delay  1000
#      --dhcp_max_ack_delay       300
#      --dhcp_max_unacked_client  300
``` 
If we add, I'm not sure if we need all the time settings for HA DHCP to work as intended. Will need to ask @cpg1111 or test it

Reply:
I don't see any issue with having an example, I think having the single dhcp container is a nice base case, but if we can reflect both, I say go for it. As to those values, as long as `dhcp2` does in fact exist and has an example of a secondary config, I think that should be good.

Reply:
Spoke to @vfrazao-ns1, we are thinking to save this for a future PR or perhaps even a separate example compose file.  We (PS) haven't had ample opportunity to test and review this as we have the other changes to these files.